docs/get-started/v2/get-started/features/recordings/live-stream-recording.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Live stream recording
+nav: 3.12
+---
+
+# Recording your live streams
+This document is specifically for recording the **HLS streams**. To learn more about 100ms recordings in general, you can refer to [this document](/get-started/v2/get-started/features/recordings/overview).
+100ms offers two major output formats for your HLS stream recordings. You can choose to record in *either* or *both* formats.
+
+<FlexContainer>
+<Callout icon="link" title="Video-on-demand (VOD) recording">
+
+Get M3U8 file which enables replayable recorded streams.
+
+[Read more →](#video-on-demand-vod-recording-m3u8)
+
+</Callout>
+
+<Callout icon="link" title="Single file per layer">
+
+Get single recording files (MP4) for different resolution layers.
+
+[Read more →](#single-recording-file-per-resolution-layer-mp4)
+
+</Callout>
+</FlexContainer>
+
+
+> **Note - External Streaming**
+> 
+> Recordings of **External Streams (RTMP-Out)** are similar to **Room Composite Recording**. They will be recorded in a single chosen resolution layer and generate a MP4 file. This recording will be a [recording asset](/server-side/v2/api-reference/recording-assets/overview#asset-types) of type `room-composite`. 
+
+
+## Video-on-demand (VOD) recording (M3U8)
+This recording is an M3U8 file (same playback format as HLS), which can be used for replaying the recorded HLS streams. This option is more suitable for Video-on-Demand use-cases as it has adaptive bitrate (ABR).
+
+You need a special player like [the one from Akamai](https://players.akamai.com/players/hlsjs) to play a M3U8 file. 
+
```

Comment:
I don't think we need to mention this anymore. We already have updated our player and users can directly play the vod from dashboard

Reply:
Resolved